external/storm-kafka/src/jvm/storm/kafka/SpoutConfig.java

```diff
@@ -29,6 +29,7 @@
 
     // setting for how often to save the current kafka offset to ZooKeeper
     public long stateUpdateIntervalMs = 2000;
+    public String emitStreamId;
```

Comment:
is it a mandatory config now? . What if users doesn't pass a emitStreamId shouldn't we use the default.